GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+# Governance of the RIOT Community
+
+The RIOT community is dedicated to creating a free and open source operating system for the constrained Internet of Things [[RFC7228], [draft-ietf-iotops-7228bis]] based on open standards.
+This document explains the governance of the project.
+
+<!-- TOC start -->
+<!-- TOC start, TOC end comments are required to generate TOC in Doxygen properly -->
+- [Values](#values)
+- [Community Processes](#community-processes)
+- [Roles](#roles)
+  + [Contributors](#Contributors)
+  + [Maintainers](#Maintainers)
+  + [Release Managers](#Release-Managers)
+  + [Admins](#Admins)
+  + [GitHub Owners](#GitHub-Owners)
+  + [Moderators](#Moderators)
+- [Decision Making](#decision-making)
+- [Meetings](#meetings)
+- [Code of Conduct](#code-of-conduct)
+- [Security Response Team](#security-response-team)
+- [Modifying this Charter](#modifying-this-charter)
+<!-- TOC end -->
+
+## Values
+
+The RIOT community embraces the following values:
+
+* **Openness:** Communication and decision-making happens in the open and is discoverable for future reference.
+  As much as possible, all discussions and work events eventually take place in public forums and open repositories.
+
+* **Fairness:** All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+* **Community over Product or Company:** Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.
+  Each contributor participates in the project as an individual.
+
+* **Inclusivity:** We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+* **Participation:** Responsibilities within the project are earned through participation, and there is a clear path up the contributor ladder into more responsible positions.
```

Comment:
I apply the patch for now. I think we can do the contribution ladder in a follow-up (see also https://github.com/RIOT-OS/RIOT/pull/21067#issuecomment-2542082020), but in case nobody picks this up, we have the text not referencing to something that's not there.